app/components/SignUp/index.jsx

```diff
@@ -101,6 +101,16 @@ export default class SignUp extends React.Component {
       });
     }
 
+    if (this.state.name.length > 25) {
+      return this.setState({
```

Comment:
I.e. we should validate the user data before adding the user to DB here: https://github.com/shri-2015-org/shrimp/blob/fix/sign-up-validation/server/db/db_core.js#L60